addons/point_of_sale/static/src/js/Screens/OrderManagementScreen/OrderFetcher.js

```diff
@@ -133,6 +133,8 @@ odoo.define('point_of_sale.OrderFetcher', function (require) {
             const idsNotInCache = ids.filter((id) => !(id in this.cache));
             if (idsNotInCache.length > 0) {
                 const fetchedOrders = await this._fetchOrders(idsNotInCache);
+                await this.comp.env.pos._loadMissingProducts(fetchedOrders);
+                await this.comp.env.pos._loadMissingPartners(fetchedOrders);
```

Comment:
Don't forget to load missing : tax, fiscal-position and pricelist to show correctly the order. 

@pimodoo I could be a good idea to save in db (at the order creation) (like sale.order), product_name, price_subtotal, price_total, tax_amount , ... to be sur that if some one modify tax, fiscal position, ... the **ticket rendering is exactly the same**. If not it could be an issue with the compliant law.

Reply:
ping @rhe-odoo